keps/sig-api-machinery/2896-openapi-v3/README.md

```diff
@@ -547,28 +585,40 @@ It's impossible to provide comprehensive guidance, but at the very high level
 These goals will help you determine what you need to measure (SLIs) in the next
 question. -->
 
+This feature should not affect the SLO of any components.
+
+The size of the OpenAPI scales as a cluster has more object types. In OpenAPI V2, aggregating the OpenAPI for all types was very resource intensive. In OpenAPI V3, aggregation is split and the aggregator will act as a proxy instead. We will monitor the current metrics to ensure that no regression is introduced with OpenAPI V3.
```

Comment:
there is an additional CRD spec generation, isn't there? What does that mean?

Reply:
Added some notes around that. There is additional overhead with V3 spec generation for all types, as well as CRD spec aggregation.